lms/static/coffee/src/instructor_dashboard/util.coffee

```diff
@@ -206,31 +227,62 @@ create_email_message_views = ($messages_wrapper, emails) ->
 
     # Create hidden section for modal window
     email_id = email_info.email['id']
-    $message_content = $('<section>', "aria-hidden": "true", class: "modal email-modal", id: "email_message_" + email_id)
+    $message_content = $(
+      '<section>',
+      "aria-hidden": "true",
+      class: "modal email-modal",
+      id: edx.HtmlUtils.interpolateHtml("email_message_{id}", {id: email_id})
```

Comment:
I'm pretty sure that JQuery will be escaping anything before it sets the attributes, so I think this could double-escape.